plugin/trino-influxdb/src/main/java/io/trino/plugin/influxdb/InfluxSplit.java

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.trino.plugin.influxdb;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.collect.ImmutableList;
+import io.trino.spi.HostAddress;
+import io.trino.spi.connector.ConnectorSplit;
+import org.openjdk.jol.info.ClassLayout;
+
+import java.util.List;
+
+import static io.airlift.slice.SizeOf.estimatedSizeOf;
+import static java.util.Objects.requireNonNull;
+
+public class InfluxSplit
+        implements ConnectorSplit
+{
+    private static final long INSTANCE_SIZE = ClassLayout.parseClass(InfluxSplit.class).instanceSize();
+
+    private final String schemaSuffix;
+    private final String tableSuffix;
+    private final String endpoint;
+    private final boolean remotelyAccessible;
+    private final List<HostAddress> addresses;
+
+    @JsonCreator
+    public InfluxSplit(@JsonProperty("endpoint") String endpoint,
+            @JsonProperty("schemaSuffix") String schemaSuffix,
+            @JsonProperty("tableSuffix") String tableSuffix)
+    {
+        this.endpoint = requireNonNull(endpoint, "endpoint is null");
+        this.schemaSuffix = schemaSuffix;
+        this.tableSuffix = tableSuffix;
+
+        remotelyAccessible = true;
+        addresses = ImmutableList.of(HostAddress.fromString(endpoint));
+    }
+
+    @JsonProperty
+    public String getEndpoint()
+    {
+        return endpoint;
+    }
+
+    @JsonProperty
+    public String getSchemaSuffix()
+    {
+        return schemaSuffix;
+    }
+
+    @JsonProperty
+    public String getTableSuffix()
+    {
+        return tableSuffix;
+    }
+
+    @Override
+    public boolean isRemotelyAccessible()
+    {
+        // only http or https is remotely accessible
+        return remotelyAccessible;
+    }
+
+    @Override
+    public List<HostAddress> getAddresses()
+    {
+        return addresses;
```

Comment:
We can pass an empty list - it should gives us the list of workers where the split should be scheduled.